identity-enabler/deviceId-mobile-app/src/pages/CreateQR.svelte

```diff
@@ -22,15 +22,21 @@
 
         try {
             const storedIdentity = await identityService.retrieveIdentity();
-            const deviceInfo = await Device.getInfo();
+            const { uuid, model, manufacturer, osVersion } = await Device.getInfo();
             credentialSubject = {
-                id: storedIdentity.doc.id,
-                deviceName: name,
-                uuid: deviceInfo.uuid,
-                manufacturer: deviceInfo.manufacturer,
-                model: deviceInfo.model,
-                operatingSystem: deviceInfo.operatingSystem,
-                osVersion: deviceInfo.osVersion
+                device: {
+                    "@context": ["https://schema.org", "https://smartdatamodels.org/context.jsonld"],
+                    type: ["Product", "Device"],
+                    id: uuid,
```

Comment:
be careful id must be a URI, you can use a URN https://datatracker.ietf.org/doc/html/rfc4122